### ISSUE TITLE
settingshelper loads the same settings lots of times

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#3810]
Dump from before:

```
➜  ka-lite git:(project-layout) ✗ bin/kalite shell
from kalite.distributed.settings import *
from kalite.django_cherrypy_wsgiserver.settings import *
from kalite.coachreports.settings import *
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.main.settings import *
from kalite.distributed.settings import *
from kalite.contentload.settings import *
from kalite.main.settings import *
from kalite.contentload.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.contentload.settings import *
from kalite.facility.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.main.settings import *
from kalite.contentload.settings import *
from kalite.facility.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.main.settings import *
from kalite.main.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.main.settings import *
from kalite.main.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.main.settings import *
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.contentload.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.contentload.settings import *
from kalite.facility.settings import *
from kalite.facility.settings import *
from kalite.playlist.settings import *
from kalite.contentload.settings import *
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.facility.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.testing.loadtesting.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.testing.settings import *
from kalite.facility.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.contentload.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.contentload.settings import *
from kalite.facility.settings import *
from kalite.facility.settings import *
from kalite.facility.settings import *
from kalite.testing.settings import *
from kalite.contentload.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.student_testing.settings import *
from kalite.store.settings import *
FAILED
from kalite.testing.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.contentload.settings import *
from kalite.contentload.settings import *
from kalite.dynamic_assets.settings import *
FAILED
from kalite.remoteadmin.settings import *
from kalite.testing.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.testing.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.ab_testing.settings import *
from fle_utils.chronograph.settings import *
from kalite.control_panel.settings import *
from kalite.inline.settings import *
FAILED
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.main.settings import *
from kalite.contentload.settings import *
from kalite.main.settings import *
from kalite.contentload.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.contentload.settings import *
from kalite.basetests.settings import *
FAILED
from kalite.django_cherrypy_wsgiserver.settings import *
from kalite.coachreports.settings import *
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.main.settings import *
from kalite.contentload.settings import *
from kalite.main.settings import *
from kalite.contentload.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.contentload.settings import *
from kalite.facility.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.main.settings import *
from kalite.contentload.settings import *
from kalite.facility.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.main.settings import *
from kalite.main.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.main.settings import *
from kalite.main.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.main.settings import *
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.contentload.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.contentload.settings import *
from kalite.facility.settings import *
from kalite.facility.settings import *
from kalite.playlist.settings import *
from kalite.contentload.settings import *
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.facility.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.testing.loadtesting.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.testing.settings import *
from kalite.facility.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.contentload.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.contentload.settings import *
from kalite.facility.settings import *
from kalite.facility.settings import *
from kalite.facility.settings import *
from kalite.testing.settings import *
from kalite.contentload.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.student_testing.settings import *
from kalite.store.settings import *
FAILED
from kalite.testing.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.contentload.settings import *
from kalite.contentload.settings import *
from kalite.dynamic_assets.settings import *
FAILED
from kalite.remoteadmin.settings import *
from kalite.testing.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.testing.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.ab_testing.settings import *
from fle_utils.chronograph.settings import *
from kalite.control_panel.settings import *
from kalite.inline.settings import *
FAILED
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.main.settings import *
from kalite.contentload.settings import *
from kalite.main.settings import *
from kalite.contentload.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.contentload.settings import *
from kalite.basetests.settings import *
FAILED
```

Nevermind about what "FAILED" means, it's harmless... just notice that things are being loaded again and again.

Now, here's the result after fixing the problem:

```
➜  ka-lite git:(project-layout) ✗ bin/kalite shell
from kalite.distributed.settings import *
from kalite.django_cherrypy_wsgiserver.settings import *
from kalite.coachreports.settings import *
from kalite.testing.settings import *
from securesync.settings import *
from fle_utils.chronograph.settings import *
from kalite.facility.settings import *
from kalite.main.settings import *
from kalite.contentload.settings import *
from kalite.main.settings import *
from kalite.testing.loadtesting.settings import *
from kalite.facility.settings import *
from kalite.main.settings import *
from kalite.main.settings import *
from kalite.playlist.settings import *
from securesync.settings import *
from kalite.facility.settings import *
from kalite.student_testing.settings import *
FAILED
from kalite.contentload.settings import *
FAILED
from kalite.remoteadmin.settings import *
from kalite.testing.settings import *
from kalite.ab_testing.settings import *
from fle_utils.chronograph.settings import *
from kalite.control_panel.settings import *
FAILED
from kalite.testing.settings import *
FAILED
```

**Comment by benjaoming** [comment]
This PR is really not very important, because I will burn settingshelper down shortly.

**Comment by benjaoming** [comment]
Remaining duplicates are because INSTALLED_APPS was being built wrongly because of weird stuff like for instance this in `testing.settings`:

```
INSTALLED_APPS = (
    'kalite.facility',  # Create users
    'kalite.main',  # Probing / creating Log objects, accessing topic_tools
    'kalite.main',  # Probing / creating Log objects
    'kalite.topic_tools',  # For benchmarking, finding random exercises
    'securesync',  # Uses securesync testing hooks
)
```

Why kalite.main two times!?


Full list of INSTALLED_APPS in old version of `kalite shell`:

```
>>> for a in settings.INSTALLED_APPS:
...     print "'%s'" % a
... 
'kalite.facility'
'kalite.main'
'kalite.main'
'kalite.topic_tools'
'securesync'
'kalite.testing.loadtesting'
'django.contrib.auth'
'django.contrib.messages'
'django.contrib.staticfiles'
'fle_utils.django_utils'
'kalite.coachreports'
'kalite.contentload'
'kalite.testing'
'kalite.i18n'
'kalite.updates'
'django.contrib.sessions'
'fle_utils.config'
'south'
'fle_utils.chronograph'
'tastypie'
'fle_utils.handlebars'
'fle_utils.backbone'
'fle_utils.build'
'kalite.control_panel'
'kalite.django_cherrypy_wsgiserver'
'kalite.playlist'
'kalite.student_testing'
'kalite.caching'
'kalite.store'
'kalite.remoteadmin'
'kalite.ab_testing'
'kalite.dynamic_assets'
'django.contrib.admin'
'django.contrib.contenttypes'
'kalite.distributed'
'compressor'
'django_js_reverse'
'kalite.inline'
'fle_utils.testing'
'kalite.basetests'
```

**Comment by benjaoming** [comment]
Will be fixed by  #3809

**Comment by benjaoming** [comment]
#3809 was merged, so closing!
